woocommerce/woocommerce-ios
# Dashboard: don't show the error toast when Jetpack site stats module is turned off

Fixes #932

## Changes
- Identified Jetpack site stats module disabled error at Yosemite layer (`SiteVisitStatsStoreError`) and to the app
- Handled `SiteVisitStatsStoreError` in `DashboardViewController`: in case of `statsModuleDisabled`, do nothing the first part for https://github.com/woocommerce/woocommerce-ios/issues/1070

## Testing
- Prerequisite: have a store with Jetpack installed
- Go to Jetpack modules settings page at your store's `/wp-admin/admin.php?page=jetpack_modules`, and find the "site stats" module
- Deactivate the "site stats" module
- Launch the app with the store --> there should not be a generic alert as in the screenshot [here](https://github.com/woocommerce/woocommerce-ios/issues/932#issuecomment-488332137)
- Go back to the Jetpack modules settings, and reactivate the "site stats" module --> the Dashboard should display the latest visitor stats

## Example screenshot

On the Dashboard (notice the toast at the bottom):

before | after (no toast)
--- | ---
![Simulator Screen Shot - iPhone SE - 2019-07-15 at 16 25 55](https://user-images.githubusercontent.com/1945542/61246633-51d03300-a71d-11e9-804d-60a4ff67ad97.png) | ![Simulator Screen Shot - iPhone SE - 2019-07-15 at 16 25 13](https://user-images.githubusercontent.com/1945542/61246636-5399f680-a71d-11e9-892a-7c52ac5cf9cb.png)


Update release notes:
